Tools/2MGFX/MonoGame.Framework/GraphicsDevice.cs

```diff
@@ -1,7 +1,18 @@
-﻿namespace Microsoft.Xna.Framework.Graphics
+﻿using System;
+
+namespace Microsoft.Xna.Framework.Graphics
 {
     // Dummy class for 2MGFX to compile.
     public class GraphicsDevice
     {
+        internal void AddResourceReference(WeakReference resourceReference)
+        {
+            
+        }
+
+        internal void RemoveResourceReference(WeakReference resourceReference)
+        {
+            
+        }
```

Comment:
I haven't dug into it to see why we need dummy classes, but I'm sure there was a good reason at the time :) Do people still use 2MGFX as a standalone executable? I guess most people just it in the content pipeline, where it's compiled in to the main content pipeline assembly.